src/node_internals.h

```diff
@@ -199,14 +199,23 @@ v8::MaybeLocal<v8::Value> InternalMakeCallback(
 
 class InternalCallbackScope {
  public:
-  // Tell the constructor whether its `object` parameter may be empty or not.
-  enum ResourceExpectation { kRequireResource, kAllowEmptyResource };
+  enum Flags {
+    // Tell the constructor whether its `object` parameter may be empty or not.
```

Comment:
nit: why not a meaningful zero value, I might add but just detail.

Reply:
As in, something like `kNoFlags = 0`? Sure, done.